src/main/resources/application.properties

```diff
@@ -44,7 +44,7 @@ uri.builder.get.file.with.patch.configuration=${PnSsUriBuilderGetFileWithPatchCo
 aruba.cert_id=${PnSsArubaCertId:AS0}
 aruba.enabled.log = ${PnSsArubaEnabledLog:false}
 #configurazione per ambiente di sviluppo
-aruba.sign.timeout=${PnSsArubaSignTimeout:20}
+aruba.sign.timeout=${PnSsArubaSignTimeout:60}
```

Comment:
Possiamo aggiungere la variabile PnSsArubaSignTimeout nel microservice.yaml in modo da poterla pilotare da configurazione?

Reply:
Viste le dipendenze con il visibilityTo della coda PnSsQueueStagingBucketStack e la stretta correlazione ai meccanismi di retry, tale configurazione verrà esternalizzata successivamente